From 558cccb43b2b3fe2e5ce03ca52902bdbcec45a3c Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:05:35 +0200
Subject: [PATCH 12/12] Merge android-9.0.0_r12

commit b7e2cae84a5f4baa7c5504ded1be842dfdd4d9b5
Merge: 6c07d73 616d275
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Jul 18 03:04:08 2018 +0000

    Snap for 4899218 from 616d275b3bf0395db7c2516423b58f9df5807017 to pi-dr1-release

    Change-Id: Ic2a609c34c0c1b7801d2af0a6896f21353984b1d

commit 616d275b3bf0395db7c2516423b58f9df5807017
Author: ChengYou Ho <chengyouho@google.com>
Date:   Wed Jul 4 01:57:58 2018 +0800

    Add control access for sdm710

    Bug: 111163725
    Change-Id: Id4a5a04b61689512735fca2eefae61d8dd94046d
    Signed-off-by: ChengYou Ho <chengyouho@google.com>

Change-Id: I2e2309b46f9bc0a6a956377860999b4039f5eb3a

# Conflicts:
#	Android.mk
---
 Android.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index c72bd9b..4123978 100644
--- a/Android.mk
+++ b/Android.mk
@@ -12,7 +12,7 @@ ifneq ($(BOARD_IS_AUTOMOTIVE),true)
       else ifneq ($(filter msm8992,$(TARGET_BOARD_PLATFORM)),)
         #For msm8992 use msm8994
         include $(call all-named-subdir-makefiles,msm8994)
-      else ifneq ($(filter msm8960 msm8084 msm8974 msm8994 msm8996 msm8998 sdm845,$(TARGET_BOARD_PLATFORM)),)
+      else ifneq ($(filter msm8960 msm8084 msm8974 msm8994 msm8996 msm8998 sdm845 sdm710,$(TARGET_BOARD_PLATFORM)),)
         #For these, use their platform name as the subdirectory
         include $(call all-named-subdir-makefiles,$(TARGET_BOARD_PLATFORM))
       else ifeq ($(filter msm8916,$(TARGET_BOARD_PLATFORM)),)
@@ -22,7 +22,7 @@ ifneq ($(BOARD_IS_AUTOMOTIVE),true)
       endif #TARGET_BOARD_PLATFORM
 
     else
-      ifneq ($(filter sdm845,$(TARGET_BOARD_PLATFORM)),)
+      ifneq ($(filter sdm845 sdm710,$(TARGET_BOARD_PLATFORM)),)
         include $(call all-named-subdir-makefiles,$(TARGET_BOARD_PLATFORM))
       else ifneq ($(filter msm8909 msm8226 ,$(TARGET_BOARD_PLATFORM)),)
         ifeq ($(TARGET_SUPPORTS_QCOM_3100),true)
-- 
2.17.1

